crm_claim/report/crm_claim_report_view.xml

```diff
@@ -70,7 +70,7 @@
     <record id="action_report_crm_claim" model="ir.actions.act_window">
         <field name="name">Claims Analysis</field>
         <field name="res_model">crm.claim.report</field>
-        <field name="view_type">form</field>
+        <field name="binding_view_types">form</field>
```

Comment:
I think you should only add `binding_view_types` in actions that in v12 has `multi = ...`

Reply:
Ok thanks!